diff --git a/install_repo.sh b/install_repo.sh
index 4d906d9..56810e8 100644
--- a/install_repo.sh
+++ b/install_repo.sh
@@ -57,18 +57,18 @@ echo "Making BLAS"
 ./apply_patches.sh patches/f2c_BLAS-3.8.0 f2c_BLAS-3.8.0
 cd f2c_BLAS-3.8.0
 cp $blas_make make.inc
-make
+make -j 4
 cd ../
 echo "Making CBLAS"
 ./apply_patches.sh patches/CBLAS CBLAS
 cd CBLAS
 mkdir -p lib
 cp $cblas_make Makefile.in
-make
+make -j 4
 cd ../
 echo "Making CLAPACK"
 ./apply_patches.sh patches/CLAPACK-3.2.1 CLAPACK-3.2.1
 cd CLAPACK-3.2.1
 mkdir -p SRC/VARIANTS/LIB
 cp $clapack_make make.inc
-make
+make -j 4
