diff --git a/src/matrix/Makefile b/src/matrix/Makefile
old mode 100644
new mode 100755
index 398179a35..c903fbfd4
--- a/src/matrix/Makefile
+++ b/src/matrix/Makefile
@@ -10,7 +10,6 @@ include ../kaldi.mk
 
 # you can uncomment matrix-lib-speed-test if you want to do the speed tests.
 
-TESTFILES = matrix-lib-test sparse-matrix-test numpy-array-test #matrix-lib-speed-test
 
 OBJFILES = kaldi-matrix.o kaldi-vector.o packed-matrix.o sp-matrix.o tp-matrix.o \
            matrix-functions.o qr.o srfft.o compressed-matrix.o \
diff --git a/src/matrix/Makefile b/src/matrix/Makefile
old mode 100644
new mode 100755
index 398179a35..eee67c9d7
--- a/src/matrix/Makefile
+++ b/src/matrix/Makefile
@@ -8,10 +8,6 @@ OPENFST_LDLIBS =
 include ../kaldi.mk
 
 
-# you can uncomment matrix-lib-speed-test if you want to do the speed tests.
-
-TESTFILES = matrix-lib-test sparse-matrix-test numpy-array-test #matrix-lib-speed-test
-
 OBJFILES = kaldi-matrix.o kaldi-vector.o packed-matrix.o sp-matrix.o tp-matrix.o \
            matrix-functions.o qr.o srfft.o compressed-matrix.o \
            sparse-matrix.o optimization.o numpy-array.o
